.deploystack/test

```diff
@@ -0,0 +1,120 @@
+# DON'T REMOVE FROM test script.
+CYAN='\033[0;36m'
+BCYAN='\033[1;36m'
+NC='\033[0m' # No Color
+DIVIDER=$(printf %"$(tput cols)"s | tr " " "*")
+DIVIDER+="\n"
+
+function get_project_id() {
+    local __resultvar=$1
+    VALUE=$(gcloud config get-value project | xargs)
+    eval $__resultvar="'$VALUE'"
+}
+
+function get_project_number() {
+    local __resultvar=$1
+    local PRO=$2
+    VALUE=$(gcloud projects list --filter="project_id=$PRO" --format="value(PROJECT_NUMBER)" | xargs)
+    eval $__resultvar="'$VALUE'"
+}
+
+# DISPLAY HELPERS
+function section_open() {
+    section_description=$1
+    printf "$DIVIDER"
+    printf "${CYAN}$section_description${NC} \n"
+    printf "$DIVIDER"
+}
+
+function section_close() {
+    printf "$DIVIDER"
+    printf "${CYAN}$section_description ${BCYAN}- done${NC}\n"
+    printf "\n\n"
+}
+
+function evalTest() {
+    local command=$1
+    local expected=$2
+
+    local ERR=""
+    got=$(eval $command 2>errFile)
+    ERR=$(<errFile)
+
+    if [ ${#ERR} -gt 0 ]; then
+        if [ "$expected" = "EXPECTERROR" ]; then
+            printf "ok \n"
+            return
+        fi
+
+        printf "expecting no error, got error='$ERR'   \n"
+        exit 1
+    fi
+
+    if [ "$got" != "$expected" ]; then
+        printf "expecting: '$expected' got: '$got'  \n"
+        exit 1
+    fi
+
+    printf "$expected is ok\n"
+}
+# END DON'T REMOVE FROM test script.
+DIR="terraform"
+get_project_id PROJECT
+gcloud services enable cloudresourcemanager.googleapis.com --project=$PROJECT
+REGION="us-central1"
+NAME="online-boutique"
+NAMESPACE="default"
+FILEPATH_MANIFEST="../release/kubernetes-manifests.yaml"
```

Comment:
So sorry! I just realized this is something I need to do myself (i.e., I need to update [this line](https://github.com/GoogleCloudPlatform/microservices-demo/blob/main/terraform/main.tf#L78) in our existing Terraform).

Reply:
**Update:**
Just created a pull-request: https://github.com/GoogleCloudPlatform/microservices-demo/pull/1145/files

Reply:
Okay.  I will... do nothing? Yeah. Let me know if I got that wrong.

Reply:
> I will... do nothing?

LOL! 😄 
Yes, that is correct.
Sorry for the confusion.

Online Boutique's Terraform now uses `kubectl apply -k ../kustomize/` instead of `kubectl apply -f ../release/kubernetes-manifests.yaml`.
See relevant pull-request: https://github.com/GoogleCloudPlatform/microservices-demo/pull/1145/files